### ISSUE TITLE
No contigs.fasta?

### ISSUE THREAD

**Issue by reymonera** [ablab/spades#470]
Hi,

I'm currently assembling multiple samples (80). I'm using the following code:

`for f1 in *_R1_001.fastq.gz
do
	f2=${f1%%_R1_001.fastq.gz}"_R2_001.fastq.gz"
	f3=${f2%%_R2_001.fastq.gz}
	~/spades.py -1 $f1 -2 $f2 -m 30 -o output_spades_$f3 --careful --cov-cutoff auto
done`

While checking the outputs that were already made, I discovered that there is no contigs.fasta file. Apparently, according to the log, I only got the following files:

```
 * Corrected reads are in "~/paired_reads/output_spades_paired_output_799_S23_L001/corrected/"
 * Paths in the assembly graph corresponding to the contigs are in "~/paired_reads/output_spades_paired_output_799_S23_L001/contigs.paths"
 * Paths in the assembly graph corresponding to the scaffolds are in "~/paired_reads/output_spades_paired_output_799_S23_L001/scaffolds.paths"
 * Assembly graph is in "~/paired_reads/output_spades_paired_output_799_S23_L001/assembly_graph.fastg"
 * Assembly graph in GFA format is in "~/paired_reads/output_spades_paired_output_799_S23_L001/assembly_graph_with_scaffolds.gfa"
```
Is the contigs.fasta file going to appear later? Why is there a final_contigs.fasta in the K127 directory? Should I use that fasta file?

Log is attached to this question:

[spades.log](https://github.com/ablab/spades/files/4304745/spades.log)

**Comment by olga24912** [comment]
Hi @sagradamarlen 

Thank you for using SPAdes and for bug report! 

It looks like we have some bug in the mismatch correction stage which is triggered when running from a path containing spaces.  In this case the problem with "Seagate Portable Drive" folder. Please, could you rerun spades from the path without spaces? 

We will fix that bug in the next SPAdes version!

Best regards, 
Olga

**Comment by reymonera** [comment]
Thanks for your help! It worked!

**Comment by Keyth-ru** [comment]
> Hi @sagradamarlen
> 
> Thank you for using SPAdes and for bug report!
> 
> It looks like we have some bug in the mismatch correction stage which is triggered when running from a path containing spaces. In this case the problem with "Seagate Portable Drive" folder. Please, could you rerun spades from the path without spaces?
> 
> We will fix that bug in the next SPAdes version!
> 
> Best regards,
> Olga

Hi, Olga. 
I'm similar problem but with scaffold.fasta file it's not in output directory, but I don't understand as you mean with  running from a path containing spaces, can you help me?

**Comment by olga24912** [comment]
Hi, @Keyth-ru 

Sorry for the long answer, I somehow missed your message. If it's relevant, could you please attach the spades.log file?

**Comment by olga24912** [comment]
It will be fixed in 3.15

**Comment by Hamlarry** [comment]
> It will be fixed in 3.15

Hi it's 2025 now, but I have the same problem, I can't find my contigs.fasta as results that should be in the folder.
Followings are my logs, it shows having produced it, but I can find it nowhere, plz help me！！！！

**Comment by Hamlarry** [comment]
* Corrected reads are in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/corrected/
 * Assembled contigs are in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/contigs.fasta
 * Assembled scaffolds are in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/scaffolds.fasta
 * Paths in the assembly graph corresponding to the contigs are in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/contigs.paths
 * Paths in the assembly graph corresponding to the scaffolds are in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/scaffolds.paths
 * Assembly graph is in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/assembly_graph.fastg
 * Assembly graph in GFA format is in /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/assembly_graph_with_scaffolds.gfa

======= SPAdes pipeline finished WITH WARNINGS!

=== Error correction and assembling warnings:
 * 0:05:47.435   976M / 4383M WARN    General                 (launcher.cpp              : 178)   Your data seems to have high uniform coverage depth. It is strongly recommended to use --isolate option.
======= Warnings saved to /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/warnings.log

SPAdes log can be found here: /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/spades.log

Thank you for using SPAdes!
ption.
======= Warnings saved to /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/warnings.log

SPAdes log can be found here: /home/zsy/fq_files_1/spades_output/12-26CRABshishuncai/spades.log

Thank you for using SPAdes!

**Comment by asl** [comment]
> > It will be fixed in 3.15
> 
> Hi it's 2025 now, but I have the same problem, I can't find my contigs.fasta as results that should be in the folder. Followings are my logs, it shows having produced it, but I can find it nowhere, plz help me！！！！

Do not hijack the closed issues. Your problem is likely different.
